package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "q-transformer",
-    "version": "4.0.0",
```

Comment:
Did you execute a `npm version minor` command for bumping this to `4.1.0`? The absence of a package-lock.json version info change in [this commit ](https://github.com/CriminalInjuriesCompensationAuthority/q-transformer/pull/76/commits/f8606f39a07a11d9ea644d1cbe3a737e37e58de3)tells me you may not have.

could you remove this commit and run the above command in it's place.

Reply:
Done, cheers